features/http_cache/tags.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Cache invalidation trough HTTP Cache tags
+  In order to have a fast API
+  As an API software developer
+  I need to store API responses in a cache
+
+  @createSchema
+  Scenario: Create some embedded resources
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I send a "POST" request to "/relation_embedders" with body:
+    """
+    {
+      "anotherRelated": {
+        "name": "Related",
+        "thirdLevel": {}
+      }
+    }
+    """
+    Then the response status code should be 201
+    And the header "Cache-Tags" should not exist
+    And "/relation_embedders,/related_dummies,/third_levels,/relation_embedders/1,/related_dummies/1,/third_levels/1" IRIs should be purged
+
+  Scenario: Tags must be set for items
+    When I send a "GET" request to "/relation_embedders/1"
+    Then the response status code should be 200
+    And the header "Cache-Tags" should be equal to "aa9e2bee5be20590f7dcc520ce2dffca,12a0c94f947a680d68bd6f65e025457d,91774d67418192a057e25dae00345572"
+
+  Scenario: Create some more resources
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I send a "POST" request to "/relation_embedders" with body:
+    """
+    {
+      "anotherRelated": {
+        "name": "Another Related",
+        "thirdLevel": {}
+      }
+    }
+    """
+    Then the response status code should be 201
+    And the header "Cache-Tags" should not exist
+
+  Scenario: Tags must be set for collections
+    When I send a "GET" request to "/relation_embedders"
+    Then the response status code should be 200
+    And the header "Cache-Tags" should be equal to "aa9e2bee5be20590f7dcc520ce2dffca,12a0c94f947a680d68bd6f65e025457d,91774d67418192a057e25dae00345572,0b51526d04eac211b0d5e93ce6b133e3,e3d156c7d2d2b9d2228071aca2aa71ca,4f4c0d1c39015017ca3e97b15c36fc16,245ca118219102ad043d75121434c707"
+
+  Scenario: Purge item on update
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I send a "PUT" request to "/relation_embedders/1" with body:
+    """
+    {
+      "paris": "France"
+    }
+    """
+    Then the response status code should be 200
+    And the header "Cache-Tags" should not exist
+    And "/relation_embedders/1,/related_dummies/1,/third_levels/1" IRIs should be purged
+
+  @dropSchema
+  Scenario: Purge item and the related collection on update
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I send a "DELETE" request to "/relation_embedders/1"
+    Then the response status code should be 204
+    And the header "Cache-Tags" should not exist
+    And "/relation_embedders/1,/relation_embedders" IRIs should be purged
```

Comment:
What's about `related_dummies` who was previously linked to this entity? They should be invalidated too